README.md

```diff
@@ -2,7 +2,6 @@
 
 [![Hex version](https://img.shields.io/hexpm/v/ex_check.svg?color=hsl(265,40%,60%))](https://hex.pm/packages/ex_check)
 [![Build status (GitHub)](https://img.shields.io/github/workflow/status/karolsluszniak/ex_check/check/master?logo=github)](https://github.com/karolsluszniak/ex_check/actions)
```

Comment:
If we only have 1 build system, the alt and logo is no longer needed to differentiate them.

Reply:
Ok, fixed.